dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/DdAndroidGradlePlugin.kt

```diff
@@ -397,10 +400,16 @@ class DdAndroidGradlePlugin @Inject constructor(
             )
             return
         }
-
+        val composeInstrumentation = ddExtension.composeInstrumentation
         project.tasks.withType(KotlinCompile::class.java).configureEach { task ->
             task.kotlinOptions.freeCompilerArgs += listOf(
-                "-Xplugin=$pluginJarPath"
+                "-Xplugin=$pluginJarPath",
+                "-P",
+                "plugin:$KOTLIN_COMPILER_PLUGIN_ID:$TRACK_VIEWS=${composeInstrumentation.trackViews.name}",
+                "-P",
+                "plugin:$KOTLIN_COMPILER_PLUGIN_ID:$TRACK_ACTIONS=${composeInstrumentation.trackActions.name}",
+                "-P",
+                "plugin:$KOTLIN_COMPILER_PLUGIN_ID:$RECORD_IMAGES=${composeInstrumentation.recordImages.name}"
```

Comment:
or maybe we can reference `DatadogKotlinCompilerPluginCommandLineProcessor.KOTLIN_COMPILER_PLUGIN_ID` directly?